service/kafka-plugin/src/test/java/com/solace/maas/ep/event/management/agent/plugin/kafka/route/delegate/KafkaRouteDelegateImplTests.java

```diff
@@ -35,8 +34,7 @@ void testGenerateKafkaTopicListingRouteList() {
                 kafkaRouteDelegate.generateRouteList(destinations, List.of(), KafkaScanType.KAFKA_TOPIC_LISTING.name(),
                 "service1");
 
-        assertThatNoException();
```

Comment:
We dont need this for these tests either because if there's an uncaught exception, it will fail the test and if there is an exception, it returns an empty list which we specifically assert.